galaxy_ng/app/api/v3/views/excludes.py

```diff
@@ -17,10 +16,9 @@
 
 
 def get_synclist_excludes(base_path):
-    """Get Synclist from distro base_path"""
+    """Get SyncList that has same name as distro base_path"""
     try:
-        repo = AnsibleDistribution.objects.get(base_path=base_path).repository
-        synclist = models.SyncList.objects.get(repository=repo, policy="exclude")
+        synclist = models.SyncList.objects.get(name=base_path, policy="exclude")
```

Comment:
I haven't given much thought to how synclists that 'include' should be handled (or if they will be supported at all).
So just wondering if there plans for that?

Reply:
there are no plans to support 'include'